src/lib/form-field/form-field.scss

```diff
@@ -77,6 +78,9 @@ $mat-form-field-default-infix-width: 180px !default;
   height: 100%;
   overflow: hidden;
   pointer-events: none;  // We shouldn't catch mouse events (let them through).
+
+  // Helps prevent text jumpiness when transforming the label.
```

Comment:
Can you provide any background on _why_ this is so?

Reply:
Changed some things so this is no longer needed